generators/client/templates/react/src/main/webapp/app/modules/account/settings/settings.tsx.ejs

```diff
@@ -151,8 +130,7 @@ export class SettingsPage extends React.Component<IUserSettingsProps, IUserSetti
                 name="langKey"
                 className="form-control"
                 label={translate('settings.form.language')}
-                onChange={this.setLangKey}
-                defaultValue={account.langKey}
+                value={account.langKey}
```

Comment:
@bertwin this mutates props which are an anti-pattern in React/Redux, or am I mistaken, coz to me it looks like it

Reply:
@deepu i believe you're mistaken. There is no 2-way binding in react. `AvField` only takes in the value, and  `AvForm` calls the function in `onValidSubmit` with the changed values.

Reply:
Yes I know that, but its my bad here. You are right. I totally forgot about avForm here. Can you resubmit the PR please. I'll merge it

Reply:
@deepu105 thanks, I made a new PR #7642